src/track.c

```diff
@@ -84,17 +85,17 @@ static u16 iq_output_mask = 0;
  */
 tracking_channel_t tracking_channel[NAP_MAX_N_TRACK_CHANNELS];
 
-/* PRN lock counter
- * A map of PRN to an initially random number that increments each time that
- * PRN begins being tracked.
+/* signal lock counter
+ * A map of signal to an initially random number that increments each time that
+ * signal begins being tracked.
  */
-static u16 tracking_lock_counters[MAX_SATS];
+static u16 tracking_lock_counters[NUM_SATS];
```

Comment:
This will need to change in future as other bands are supported, as the lock counters are per signal, not per satellite.